ui/widgets/nuxeo-uploader-behavior.js

```diff
@@ -476,6 +476,21 @@ export const UploaderBehavior = {
   _drop(e) {
     this.toggleClass('hover', false, this._dropZone);
     e.preventDefault();
-    this.uploadFiles(e.dataTransfer.files);
+    const allFiles = e.dataTransfer.items;
+    const files = [];
+    if (allFiles.length > 0) {
+      for (let i = 0; i < allFiles.length; i++) {
+        if (allFiles[i].webkitGetAsEntry().isDirectory) {
+          this.notify({ message: this.i18n('foderUpload.error') });
```

Comment:
So, if we have multiple folders we will perform the notify several times, which doesn't seem very useful. What do you think about storing this check in a local variable, and then issue the notify later if at least one directory was found?

```js
    const allFiles = e.dataTransfer.items;
    const files = [];
    let hasFolder = false;
    if (allFiles.length > 0) {
      for (let i = 0; i < allFiles.length; i++) {
        if (allFiles[i].webkitGetAsEntry().isDirectory) {
          hasFolder = true;
        } else {
          files.push(e.dataTransfer.files[i]);
        }
      }
     if (hasFolder) {
        this.notify({ message: this.i18n('folderUpload.error') });
     }
```

Reply:
Up to discussion about this ofc. Thank you.

Reply:
And don't forget to update the key to `folderUpload.error`.